File: MES_EQUIPMENT/Models/MoldingDto.cs
public class MoldingDto
{
    public int Id { get; set; }                       // PK
    public double MoldTemp { get; set; }              // 금형 온도
    public double InjectionPressure { get; set; }     // 사출 압력
    public double CureTime { get; set; }              // 경화 시간
    public double ClampForce { get; set; }            // 클램프 힘

    // ✅ 역직렬화: byte[] → 공정 DTO
    public static MoldingDto FromBytes(byte[] payload)
    {
        int offset = 0;
        MoldingDto dto = new MoldingDto();

        dto.Id = BitConverter.ToInt32(payload, offset); offset += 4;
        dto.MoldTemp = BitConverter.ToDouble(payload, offset); offset += 8;
        dto.InjectionPressure = BitConverter.ToDouble(payload, offset); offset += 8;
        dto.CureTime = BitConverter.ToDouble(payload, offset); offset += 8;
        dto.ClampForce = BitConverter.ToDouble(payload, offset); offset += 8;

        return dto;
    }
}

File: MES_SPRING/src/main/java/com/hm/mes_final_260106/dto/MoldingDto.java
package com.hm.mes_final_260106.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class MoldingDto {

    private Long id;                // PK: 공정 ID
    private Long productionLogId;  // pk: ProductionLog 참조 (숫자만)
    private double moldTemp;        // 몰드 온도 (℃)
    private double injectionPressure; // 수지 주입 압력 (bar)
    private double cureTime;        // 경화 시간 (s)
    private double clampForce;      // 클램프 압력 (kN)
}